src/consts/warpRoutes.yaml

```diff
@@ -1,1034 +1,146 @@
+# yaml-language-server: $schema=../schema.json
 tokens:
-  - chainName: blast
+  - chainName: arbitrum
     standard: EvmHypXERC20
-    name: Renzo Restaked ETH
+    decimals: 18
     symbol: ezETH
     logoURI: https://renzo.hyperlane.xyz/logos/ezeth.svg
```

Comment:
Do these load properly? I'd expect them to fail due to the CSP security header. They're meant to be relative URIs